src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,30 @@
+package task;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a task list.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Initiliazes an empty task list.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Initializes task list with loaded tasks.
+     *
+     * @param tasks is the task list
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public ArrayList<Task> getTasks() {
```

Comment:
It seems like every other class that uses a `TaskList` needs to call this method to do their operations. Consider providing those methods (e.g. `add(Task)`) on the `TaskList` instead of allowing unrestricted manipulation of the `ArrayList`.